src/main/justin/Justin.java

```diff
@@ -0,0 +1,157 @@
+import java.io.File;
+import java.util.Scanner;
+
+public class Justin {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+
+    public Justin(String filePath) {
+
+        ui = new Ui();
+        this.storage = new Storage(filePath);
+        tasks = storage.loadFile(filePath);
+
+    }
+
+    public void run() {
+
+        /*
+         * Justin is a chatbot that help users plan and organise tasks
+         * Justin stands for JUSt a TImetable(New) : JUSTIN
+         *
+         * Justin is able to create and manage new tasks
+         *   1) Mark tasks off as done
+         *   2) Set tasks as To Do's - <keyword> <name>
+         *   3) Set tasks as Deadline's - keyword  : <keyword> <name> /by <day>
+         *   4) Set tasks as Event's - keyword : <keyword> <name> /at <day time>
+         *   5) Supports deletion of completed tasks with command delete <int>
+         *
+         * Justin also supports viewing the entire tasks list with the command list - keyword : <keyword>
+         * To end off the session user can input bye to terminate program - keyword : <keyword>
+         *
+         * @author Goh Wei Kiat aka github : mrweikiat
+         * @version CS2103T AY20/21 Semester 2, Individual Project 'IP'
+         */
+
+
+        Scanner sc = new Scanner(System.in);
+
+        // Starting line for UI
+        ui.showWelcomeMessage();
+        ui.showHelpMessage();
+
+        // Condition for Duke to stop
+        boolean terminate = false;
```

Comment:
Should this be canTerminate? Might be better to change to reflect boolean variable name